plugin/plugin.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/gaia-pipeline/gaia"
 	"github.com/gaia-pipeline/gaia/security"
 	"github.com/gaia-pipeline/gaia/workers/scheduler"
-	"github.com/gaia-pipeline/protobuf"
+	proto "github.com/gaia-pipeline/protobuf"
```

Comment:
Huh? :O

Reply:
This has been auto corrected by `goimports`. Shouldn't be a problem in my opinion.